testing/networks/80069/genesis.json

```diff
@@ -126,7 +126,8 @@
       },
       "feature": {
         "vote_extensions_enable_height": "0",
-        "pbts_enable_height": "1"
+        "pbts_enable_height": "1",
+        "sbt_enable_height": "7768335"
```

Comment:
Note: this PR is here to document a patch for handling consensus parameters update correctly and avoid Bepolia nodes crash.
https://github.com/berachain/beacon-kit/pull/2870 is the full fix for the issue, so this PR can be closed down as soon as a release containing https://github.com/berachain/beacon-kit/pull/2870 is cut.